package.json

```diff
@@ -8,7 +8,7 @@
 		"production": "pnpm exec nx run-many --nx-bail --target=production --parallel 1000 --exclude @lix-js/server --exclude @inlang/badge --exclude @inlang/manage --exclude @inlang/editor --exclude @inlang/telemetry-proxy --output-style=stream",
 		"build": "pnpm exec nx run-many --nx-bail --target=build --parallel",
 		"dev:lix": "pnpm --filter @inlang/env-variables build && pnpm --stream --parallel --filter @lix-js/* --filter @inlang/server dev",
-		"dev:lixe-fink": "pnpm --filter @inlang/env-variables build && pnpm --stream --parallel --filter @lix-js/* --filter @inlang/serve --filter @inlang/manage --filter @inlang/editor --filter @inlang/website dev",
+		"dev:lix-fink": "MOCK_TRANSLATE=true pnpm --filter @inlang/env-variables build && pnpm --stream --parallel --filter @lix-js/* --filter @inlang/server --filter @inlang/manage --filter @inlang/editor --filter @inlang/sdk --filter @inlang/website --filter @inlang/telemetry-proxy dev",
```

Comment:
If you just want to run Fink with this command, you can leave out the server and the website.

Reply:
The MOCK_TRANSLATE env var at the start actually doesn't work as intended so i'll remove that.
The rest is useful for sdk and lix development so I'll keep those for now.